examples/AN00162_i2s_loopback_demo/README.rst

```diff
@@ -21,16 +20,15 @@ loopback all 8 channels.
 Required tools and libraries
 ............................
 
- * xTIMEcomposer Tools - Version 14.3.2
- * XMOS |I2S|/TDM library - Version 2.4.0
- * XMOS GPIO library - Version 1.1.0
- * XMOS |I2C| library - Version 5.0.0
+ * xTIMEcomposer Tools - Version 15.3.0
+ * XMOS |I2S|/TDM library - Version 6.0.0
+ * XMOS Board Support library - Version 1.0.0
```

Comment:
I'm wondering if having a list with version numbers is a good idea here for maintainability. This version of lib_board_support doesn't match the version in deps.cmake.

Reply:
removed version numbers. I think this will move to xmosnotes in time.